### PR TITLE
Remove useGenericCheckout test

### PR DESCRIPTION
**Pull request by jamesgorrie**
Removes references to the useGenericCheckout test. 

This will remain in draft until we're happy with the numbers and sure that the new checkout is good.

It is good to review though.

## All tests 🍏 

Screenshot 2024-09-10 at 10 56 37

### CODE REVIEW

File: support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx [hunk 1]
```diff
@@ -392,65 +387,6 @@ export function ThreeTierLanding({
...
-				contributionCurrency,
-			);
-			/**
-			 * Lower & middle tier track component click fired via redux side effects.
-			 * Top tier accessed via network request to GuardianWeekly landing page
-			 * therefore tracking required
-			 **/
-			trackComponentClick(
-				`npf-contribution-amount-toggle-${countryGroupId}-${contributionType}-${price}`,
-			);
```
Comment (jamesgorrie) [inline support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx:417]: @GHaberis should this be fired on the pageview of the checkout?

**Review by paul-daniel-dempsey — COMMENTED** [review]
Nice simplication for `ThreeTierLanding`, reduces the amount of test too. Dont thnk the Tier3 tests are passing at the moment however.

File: support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx (hunk 1 above)
Comment (GHaberis) [reply support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx:430]: @jamesgorrie `sendEventContributionCartValue` reports to QM the "cart value" so they can attribute financial impacts of issues they pick-up, it should therefore be fired on the pageview of the checkout.

To be honest, I'd look at whether this is easily implementable and if not proceed this PR without implementing this and create a card to revisit `sendEventContributionCartValue`  as it's not a piece of data we're using - so is not a priority.

Comment (jamesgorrie) [reply support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx:430]: I'd agree that we should should look at writing these methods with the new product model.

**Review by GHaberis — APPROVED** [review]
Looks great 👍

**Pull request merged**